example/llrt-sam/template.yaml

```diff
@@ -0,0 +1,53 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  llrt-sam
+
+  Sample SAM Template for llrt-sam
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+
+  LlrtLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      ContentUri: ../../llrt-lambda-x64.zip
```

Comment:
ARM is actually faster, so we should default to that

Reply:
Also remember set `CompatibleArchitectures` to `arm64`

Reply:
Wasn't any faster when I tried arm64 https://github.com/kaihendry/llrt-aws-sam-example/actions/runs/7907126599/job/21583462311#step:7:20 vs x64 https://github.com/kaihendry/llrt-aws-sam-example/actions/runs/7907082851/job/21583326692#step:7:20 thought it probably wasn't the best test

Reply:
That probably due to caches, Lambda uses a tiered cache that gets a bit better after a while. Also ARM is lower cost.  It's not a major difference in performance but you should experience slightly faster cold starts at least (at lower cost) :)